examples/test_field_sweep.py

```diff
@@ -0,0 +1,81 @@
+import pyspecdata as psd
+import pyspecProcScripts as prscr
+import numpy as np
+import matplotlib.pyplot as plt
+
+signal_pathway = {"ph1": 1}
+with psd.figlist_var() as fl:
+    for thisfile, exptype, nodename, post_proc, label_str in [
+        (
+            "240924_13p5mM_TEMPOL_field.h5",
+            "ODNP_NMR_comp/field_dependent",
+            "field_1",
+            "field_sweep_v4",
+            "9-24-24 13.5 mM TEMPOL sweep",
+        )
+    ]:
+        s = psd.find_file(
+            thisfile,
+            exp_type=exptype,
+            expno=nodename,
+            postproc=post_proc,
+            lookup=prscr.lookup_table,
+        )
+        s_indirect = s["indirect"]
+        ###{{{ Raw data
+        label = label_str
+        fl.basename = "(%s)" % label
+        fl.next("raw data")
+        fl.image(s, interpolation="auto")
+        ###}}}
+        ###{{{ Time Domain
+        field_axis = []
+        for indir_val in range(np.shape(s.getaxis("indirect"))[0]):
+            field_axis.append(s.getaxis("indirect")[indir_val][0])
```

Comment:
this is not good b/c it's not `vectorized` code (look that up)

I am pretty sure that the axis is a `structured array` and that you're trying to pull a particular `field` -- you can see `proc_raw.py` for an example of how we do this.

Reply:
here the words that are in `this format` (aside from the file name) are "vocab words" that you can look up  -- all of them are relevant to numpy